content/en/blog/_posts/non-root-containers-and-devices.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: 'Non-root Containers And Devices'
+date: 2021-09-YY
+slug: non-root-containers-and-devices
+---
+
+**Author:** Mikko Ylinen (Intel)
+
+## Abstract
+One of the key security principles for running containers in Kubernetes is the
+principle of least privilege. The Pod/container `securityContext` specifies the config
+knobs to set, e.g., Linux* capabilities, MAC policies, and user/group ID values to achieve this.
+
+Furthermore, the cluster admins are supported with tools like [Pod Security Policy (deprecated)](docs/concepts/policy/pod-security-policy/) or
+[Pod Security Admission](docs/concepts/security/pod-security-admission/) to enforce the desired security settings for pods that are being deployed in
+the cluster. These settings could, for instance, require that containers must be `runAsNonRoot` or
+that they are forbidden from running with root's group ID in `runAsGroup` or `supplementalGroups`.
+
+However, the user/group ID related security settings trigger a problem when users want to
+deploy containers that use accelerator devices (via [Kubernetes Device Plugins](docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/)) on Linux. In this blog
+post we talk about the problem in detail and describe the work that was made so far to address it. It's not meant to be a long story about getting the ["k/k issue"](https://github.com/kubernetes/kubernetes/issues/92211) fixed but to raise the awaress of it and important device use-cases too. This is needed as we work on new related features such as support for user namespaces.
```

Comment:
The purpose/goal of the post needs to be near the beginning, under the title. Readers need to know what the blog is about before they start reading it. I'd recommend summarizing the Problem Statement into one or two sentences, then saying "in this blog post we describe blah", right at the top of the page.